ods_ci/tests/Tests/650__distributed_workloads/test-distributed-workloads-metrics-ui.robot

```diff
@@ -0,0 +1,115 @@
+${PRJ_TITLE}=    test-dw-ui
+${PRJ_TITLE_NONADMIN}=    test-dw-nonadmin
+${PRJ_DESCRIPTION}=    project used for distributed workload metrics
+
+
+*** Test Cases ***
+Verify Workload Metrics Home page Contents
+    [Documentation]    Verifies "Workload Metrics" page is accessible from
+    ...                the navigation menu on the left and page contents
+    [Tags]    Sanity
+    ...       DistributedWorkloads
+    Open Distributed Workload Metrics Home Page
+    Wait until Element is Visible    ${DISTRIBUITED_WORKLOAD_METRICS_TEXT_XP}   timeout=20
+    Page Should Contain Element     ${DISTRIBUITED_WORKLOAD_METRICS_TITLE_XP}
+    Page Should Contain Element     ${DISTRIBUITED_WORKLOAD_METRICS_TEXT_XP}
+    Page Should Contain Element     ${PROJECT_XP}
+    Page Should Contain Element     ${PROJECT_METRICS_TAB_XP}
+    Page Should Contain Element     ${WORKLOAD_STATUS_TAB_XP}
+    Click Element    ${REFRESH_INTERNAL_MENU_XP}
+    ${get_refresh_interval_list}=    Get All Text Under Element   xpath=//*[starts-with(@id, "select-option-")]
+    Lists Should Be Equal    ${REFRESH_INTERNAL_LIST}    ${get_refresh_interval_list}
+
+Verify Project Metrics Default Page contents
+    [Tags]    Sanity
+    ...       DistributedWorkloads
+    [Documentation]    Verifiy Project Metrics default Page contents
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE}
+    Check Project Metrics Default Page Contents
+
+Verify Distributed Workload status Default Page contents
+    [Tags]    Sanity
+    ...       DistributedWorkloads
+    [Documentation]    Verifiy distributed workload status page default contents
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE}
+    Check Distributed Workload Status Page Contents
+
+Verify That Not Admin Users Can Access Distributed workload metrics default page contents
+    [Documentation]    Verify That Not Admin Users Can Access Distributed workload metrics default page contents
+    [Tags]    Sanity
+    ...       DistributedWorkloads
+    Launch Dashboard    ocp_user_name=${TEST_USER_3.USERNAME}    ocp_user_pw=${TEST_USER_3.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER_3.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}
+    ...    browser=${BROWSER.NAME}    browser_options=${BROWSER.OPTIONS}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE_NONADMIN}   description=${PRJ_DESCRIPTION}
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE_NONADMIN}
+    Check Project Metrics Default Page Contents
+    Check Distributed Workload Status Page Contents
+    [Teardown]    Run Keywords
+    ...    Delete Data Science Project   ${PRJ_TITLE_NONADMIN}
+    ...    AND
+    ...    Wait Until Data Science Project Is Deleted  ${PRJ_TITLE_NONADMIN}
+    ...    AND
+    ...    Switch Browser    1
```

Comment:
Shouldn't this be part of `Project Suite Teardown` rather than being part of the test itself?

Reply:
We can delete all the created Projects in `Project Suite Teardown `using method `Delete Data Science Projects From CLI` but thinking  if we ran tests independently this project may not be available and test fails

Reply:
ok